docs/conversion_examples_gallery/recording/tdt.rst

```diff
@@ -19,7 +19,7 @@ Convert TDT data to NWB using :py:class:`~neuroconv.datainterfaces.ecephys.tdt.t
     >>> # For this data interface we need to pass the folder_path with the location of the data
     >>> folder_path = f"{ECEPHY_DATA_PATH}/tdt/aep_05"
     >>> # Change the folder_path to the location of the data in your system
-    >>> interface = TdtRecordingInterface(folder_path=folder_path, verbose=False)
+    >>> interface = TdtRecordingInterface(folder_path=folder_path, gain=1.0, verbose=False)
```

Comment:
@CodyCBakerPhD maybe we should add a note to users on where or how to get this? 

It could also be in the docstring of the class.

Reply:
Aside from justifying our choice of `1.0` (which I thought Szonja did) we have no instructions for how to find - assumption is user knows where/who they bought it from and would be able to consult their docs or get in touch with manufacturer